docs/4.4/api-reference.md

```diff
@@ -796,14 +798,114 @@ if err := client.DeleteAccessRequest(ctx, accessReqID); err != nil {
+
+You may want to use this API to manage your CA if:
+
+- You don't want to automate rotating certificates with `tctl` ([rotating CA with tctl](admin-guide.md#certificate-rotation))
+- You want to set up a custom system for rotating certificates for more security and stability
+- You want to rotate certificates in a trusted cluster without manually fetching the new certs on remote clusters
+
+### Certificate Rotation
+
+To maintain security across your cluster, it is a good idea to set up automatic [certificate rotation](architecture/authentication.md#certificate-rotation). The most fault tolerant, fast, and effective way to rotate the CA is to use a custom solution, but we'll start with the default automated solution and work from there.
+
+You can use `tctl auth rotate` or the RPC below to start the rotation in `auto` mode. You can also provide a custom grace period, or target only the host/user CA with both the RPC and `tctl auth rotate --type=user --grace-period=10h`. type is useful if you want to rotate the user and host CA's with different strategies.
 
 ```go
-ca, err := client.GetCertAuthority(services.CertAuthID{
-  DomainName: clusterName,
-  Type:       services.HostCA,
-}, false)
-if err != nil {
+// This will start the an automatic rotation that schedules each phase of the rotation in equal increments.
+gracePeriod := time.Hour * 24
+req := auth.RotateRequest{
+  Mode: services.RotationModeAuto
+  GracePeriod: &gracePeriod // defaults to 48 hours
+  Type: services.UserCA // Leave empty to target UserCA and HostCA
+}
+if err := client.RotateCertAuthority(req); err != nil {
+  return err
+}
+```
+
+**Rotation Phases**
+
+In `auto` mode, the rotation is scheduled to run in a series of phases. These phases occur in the order below, unless you set a custom schedule, or run each phase manually.
+
+1. `Standby`: No rotation operations underway.  This is the beginning and end state of every CA rotation. If a CA's rotation is not in the standby state, a new rotation cannot begin.
+2. `Init`: New Certificate Authority is issued, but it remains unused while users and servers get updated certificates.
+3. `Update Clients`: Client credentials will have to be updated and reloaded, but servers will still use and respond with old credentials (grace period).
+4. `Update Servers`: Servers will have to reload and should start serving TLS and SSH certificates signed by new CA (retrieved in previous phase).
+5. `Rollback`: Rolls the CA back to the old CA. This phase is not triggered automatically, but can be used in the case of a rotation failure. e.g. if your servers go down during the rotation, and aren't up in time within the grace period, leaving them with only the old certificates.
+
+!!! Note
+    It is necessary for remote trusted clusters to fetch the new certificate authority manually.
```

Comment:
Can you outline these manual steps?

Reply:
@klizhentas I saw this comment in the code around `RotationPhaseInit`, but I'm not sure what the best steps are for a remote cluster to retrieve the new certs.

RotationPhaseInit = is a phase of the rotation when new certificate authority is issued, but not used **It is necessary for remote trusted clusters to fetch the new certificate authority, otherwise the new clients will reject it**

I couldn't find any more information on this in the code or the docs, so I'm not sure if I misinterpreted the comment, or if it is still true.

Reply:
I removed the note since I misinterpreted the comment.